crates/iroha_data_model/src/block.rs

```diff
@@ -72,7 +72,7 @@ mod model {
     )]
     #[display(fmt = "({header})")]
     #[allow(missing_docs)]
-    pub(crate) struct BlockPayload {
+    pub struct BlockPayload {
```

Comment:
I’m not sure how to address the visibility of `BlockPayload`

![image](https://github.com/user-attachments/assets/634934cd-33b5-4f01-993f-cafdb4fc310d)

Reply:
I would suggest keep it `pub(crate)` and add `#[allow(clippy::redundant_pub_crate)]`

Related: https://github.com/rust-lang/rust-clippy/issues/13676

Reply:
Reverted visibility of `BlockPayload` and `TransactionPayload`